specify/specify7
# Rescope upload tree when there is only 1 tree in the upload plan

Fixes #5338

The upload tree wasn't being rescoped in cases where upload plan had only 1 tree, due to which upload would be scoped to the default tree of the collection which meant upload didn't work if the upload tree of the plan was not the same as the default tree



### Checklist

      and self-explanatory (or properly documented)

### Testing instructions



- Go to the workbench
- Create an upload plan with only 1 tree which isn't the original tree
- [ ] Verify upload works
